src/index.html

```diff
@@ -11,6 +11,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="bigger__search search"
```

Comment:
i think it's more clear to set generic class first and then override it by modifier, not otherwise